testsuite/src/test/java/org/mvcspec/ozark/test/ThymeleafIT.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © 2017 Ivar Grimstad (ivar.grimstad@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.mvcspec.ozark.test;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.asset.FileAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.shrinkwrap.resolver.api.maven.Maven;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.net.URL;
+import java.nio.file.Paths;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author Gregor Tudan
+ */
+@RunWith(Arquillian.class)
+public class ThymeleafIT {
+
+    private static final String WEB_INF_SRC = "src/main/resources/thymeleaf/";
+
+    @ArquillianResource
+    private URL baseURL;
+
+    @Drone
+    private WebDriver webDriver;
+
+    @Deployment(testable = false, name = "thymeleaf")
+    public static Archive createDeployment() {
+        return ShrinkWrap.create(WebArchive.class)
```

Comment:
That looks great! But I wonder if we should create some util class which simplifies building a "common base archive". I did this for the TCK, see here:

https://github.com/mvc-spec/mvc-tck/blob/master/tests/src/main/java/org/mvcspec/tck/tests/mvc/models/BuiltinEngineModelTest.java#L48

The implementation is simple, as it delegates all the work to a custom archive builder:

https://github.com/mvc-spec/mvc-tck/blob/b79c16bab5cb8c035a1415d55c5551d67ddfd776/tests/src/main/java/org/mvcspec/tck/util/Archives.java#L28-L34

And a custom builder with specialized methods to build MVC archives:

https://github.com/mvc-spec/mvc-tck/blob/master/tests/src/main/java/org/mvcspec/tck/util/WebArchiveBuilder.java

It is not really required to have something like this in the first PR, but I guess it would make sense to do this shortly after merging it, so we have an easy job of migrating the other examples.